docs/source/index.rst

```diff
@@ -57,10 +57,10 @@ User Documentation
     :maxdepth: 2
 
     quickstart.rst
+    recipes.rst
     specification.rst
     distributing-files.rst
     key-value-store.rst
-    recipes.rst
```

Comment:
I'd rather keep `recipes` where it was. They can be used, but aren't nearly as useful as people writing their own. They're also not necessarily the best uses of skein (in particular, there are better ways to deploy notebooks than this).